private/pkg/storage/errors.go

```diff
@@ -41,6 +44,21 @@ func IsNotExist(err error) bool {
 	return errors.Is(err, errNotExist)
 }
 
+// NewErrInvalidPath returns a new error for an invalid path.
```

Comment:
What is an invalid path? I don't understand this from an API perspective, ie this doesn't make sense in the context of paths and buckets within the storage API.

Reply:
Commented - just removing this and returning is-not-exist in the case of non-regular file. storageos Buckets don't iterate over non-regular files in Walk as it is, it seems consistent to treat this as a not-exist case.